devfiles/apache-camel-k/meta.yaml

```diff
@@ -0,0 +1,6 @@
+---
+displayName: Apache Camel K
+description: Stack with tooling ready to develop Integration projects with Apache Camel K
+tags: ["Apache Camel K", "Red Hat Fuse", "Integration"]
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+globalMemoryLimit: 2930Mi
```

Comment:
the globalMemoryLimit has not been changed for existing stack https://github.com/eclipse/che-devfile-registry/blob/6b8dcf222e2e7d35933592582282ec8e7b6e7730/devfiles/apache-camel-springboot/meta.yaml#L6
Does it mean that Apache Camel SpringBoot stack won't work on che.openshift.io anymore after upgrade to 7.5.0?

Reply:
(I updated to 2850Mi in this PR for the Apache Camel K stack)

Reply:
The globalMemoryLimit is just a workaround, since Che currently cannot calculate the limit from a list of plugins. I believe the current stack works on che.openshift.io without issue -- the globalMemoryLimit is more a suggestion than a requirement :)

I believe workspaces start even if it's set to `10Gi`, or missing entirely.